config/samples/perses.dev_v1alpha1_perses_client_tls.yaml

```diff
@@ -0,0 +1,58 @@
+  ca.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURBekNDQWV1Z0F3SUJBZ0lVSmk1THpybTdkRHdFUHhZbE9kY1FtQldVb0VFd0RRWUpLb1pJaHZjTkFRRUwKQlFBd0VURVBNQTBHQTFVRUF3d0djR1Z5YzJWek1CNFhEVEkxTURJeU5qQXdORFV3TTFvWERUTTFNREl5TkRBdwpORFV3TTFvd0VURVBNQTBHQTFVRUF3d0djR1Z5YzJWek1JSUJJakFOQmdrcWhraUc5dzBCQVFFRkFBT0NBUThBCk1JSUJDZ0tDQVFFQXJVdlZ1enlKc2NucCtEWnVoZ2hZTDVwc3Z1amIzTUxCOG9YRnhSQllkMEl1V0t5YjZLLzIKTWlMQmN6dXh5UDliVzhVSGdRUXRua1dBWmtrUjlkd05ieERqUlRDeU0zWHY1aU9KWkZ4WlBINUF0bE1qRGx3MwpGb3dTNytWWXkwZ3ZZN3ZCQkVUUXQ4d2N3Z1FrdUdQdlFDalBVb0lWT2dBV2ZiMzN3NzU1MWUzOE9SMDdjcVdICisvUmpkZllxcUhkZEZ1WWtJMW5Ua0lybHpBVW9DeisyL3ZLQUgwTjJYVVlKUFhXV29qUCs3bDhRM0tlRDJ4RS8KbDc2Q0hRMmxoSWRBNkIwbXdlN1ROaUVYb1FsTnFPV1g3bVRoOWl5SElUNzdNQzI3dVhEZTAzT0djblZ3eUJuYQpwOFh5aUxaZGdQOUwxUlZOUitPaTNiOUJrTWE3L3BrbEF3SURBUUFCbzFNd1VUQWRCZ05WSFE0RUZnUVVITVNvCjd0OFhkWlZZY0Q2WTdaeGVLcnc4c2NNd0h3WURWUjBqQkJnd0ZvQVVITVNvN3Q4WGRaVlljRDZZN1p4ZUtydzgKc2NNd0R3WURWUjBUQVFIL0JBVXdBd0VCL3pBTkJna3Foa2lHOXcwQkFRc0ZBQU9DQVFFQVl6c1RRZUlFUElYWApwL0NJOEExK3AwbVE1WE5wdFo0VHZWYmNwNVhKWTMwNzBJSy9POEM0Sk5KV0lER2QyV0VLMno2ZHBxL0VQTW42CmhiRlhjMGlHbVJEaUhrUHQvUDgzN0llQUJtdjR4UVk5SzhWNVJ2WXo4U09GQkV6OUFQQW4zNHM0RXgyS0xiWVoKWVJONXFRclFxaE9OenNzc1d1YlBYZ0dKNkdMcXk1eDJpeXZKWDBiTUJ2NmcxZnRINk5IZWN1UFVYZjZjdnFXegpUdTZ2b25TaXUrN0o2SVg1dyt1aGFFRXhsYW5qS3ExNDZpUHoxZnJMK3NYRzBMYXh5bStUcnVDOEZVZWp5SXhQCkZRdW9uS3BQMjZEQTZraWdSck1wQWd4NGxEY3ZvS0c1MFY3UmRmcTNPZWc4VTd2VXl5UlRrbGkya2tDS20vRWMKZ1NJNzFXbGR6QT09Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
+---
+apiVersion: perses.dev/v1alpha1
+kind: Perses
+metadata:
+  labels:
+    app.kubernetes.io/name: perses
+    app.kubernetes.io/instance: perses-tls-sample
+    app.kubernetes.io/part-of: perses-operator
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: perses-operator
+  name: perses-tls-sample
+  namespace: perses-dev
+spec:
+  client:
+    tls:
+      enable: true
+      caCert:
+        type: secret
+        name: perses-certs
+        certFile: ca.crt
+      userCert:
+        type: secret
+        name: perses-certs
+        certFile: tls.crt
+        certKeyFile: tls.key
+
+  config:
+    database:
+      file:
+        folder: "/perses"
+        extension: "yaml"
+    schemas:
```

Comment:
if you are loading the last version of Perses (v0.51.0-beta.0) the schemas config should be removed as it is not used anymore and will trigger un warning saying it is deprecated